frontend/test/proxy.js

```diff
@@ -23,7 +23,7 @@ const talkback = require("talkback")
 /** @typedef {ReturnType<TalkbackOptions['tapeDecorator']>} Tape */
 
 const port = 49153
-const host = "https://api.openverse.engineering"
+const host = "https://api.openverse.org"
```

Comment:
I wonder if this should use the env variable...

Reply:
Could do! Good for another issue, I think though.